apps/server/src/utilities/generateEncodedString.ts

```diff
@@ -0,0 +1,14 @@
+import { promisify } from 'util';
+import jwt, { Secret, SignOptions } from 'jsonwebtoken';
+
+export const generateEncodedString = async (
+    object: object,
+    secretKey: string,
+    expiresIn: string,
+): Promise<any> => {
```

Comment:
This util should return a string not any. If you have problems with the promisify return type, try this. 
`return await promisify<object, Secret, SignOptions>(jwt.sign)(
        object,
        secretKey,
        { expiresIn },
    ) as unknown as string;`